raspyrfm_client/device/manufacturer/intertechno/IT1500.py

```diff
@@ -0,0 +1,50 @@
+from raspyrfm_client.device import actions
+from raspyrfm_client.device.base import Device
+
+
+class IT1500(Device):    
+    _connair_params = {'repetitions': 6, 'pauselen': 11125, 'steplen': 275}
```

Comment:
true, true.... but not all parameters of those 3 are fixed per device. There are devices which need more repetition for PAIR action. My suggestion is: change return type of method get_pulse_data. It could return:
repetitions, pauselen, steplen and pulse data all together. What do you think?